src/components/PassageMatches.tsx

```diff
@@ -17,7 +17,7 @@ const PassageMatches = ({ passages, setPassageIndex, activeIndex }: TProps) => {
       ) : (
         <div className="divide-lineBorder divide-y passage-matches-list">
           {passages.map((item, index: number) => (
-            <div key={item.text_block_id}>
+            <div key={item.text_block_id} data-analytics-passage={index + 1}>
```

Comment:
Just need to be extra careful here as you are going from zero-based to one-based

Reply:
This is to make it "friendly" for the analytics guys to handle